api/swagger.yaml

```diff
@@ -243,6 +243,9 @@ definitions:
       ReadOnly:
         description: "Whether the mount should be read-only."
         type: "boolean"
+      Consistency:
+        description: "The consistency requirement for the mount: `default`, `consistent`, `cached`, or `delegated`."
```

Comment:
I'm not sure what the API semantics are regarding omitted fields but the `default` mode is present in the `--mount` long-form CLI so that every mode is representable as an explicit value. At least in that context, having `default` allows things like scripting `consistency=$MODE` where the `MODE` variable can then be set to any of `default`, `delegated`, `cached`, or `consistent`. Without a representation for every mode, there is not a nice way to set the consistency to `default` without additional complexity of omitting `consistency=` sometimes.